charts/sm-operator/.helmignore

```diff
@@ -0,0 +1,28 @@
+# Patterns to ignore when building packages.
```

Comment:
Maybe in this ignore file we can clean up the spacing a bit? Like an empty line after comments or something to make it a bit more readable.

Also between llines 23 and 26 do we need two break lines?

Reply:
Please check again.